synapseclient/__main__.py

```diff
@@ -55,20 +55,21 @@
 A few more commands (cat, create, update, associate)
 
 """
+
```

Comment:
let's follow pipenv's convention for imports:
use `import` for external imports. For imports w/ lots of `.` alias it with `_` (e.g. `import urllib3.util as urllib3_util`)
for local imports, use relative paths and `from`